commands.js

```diff
@@ -2956,6 +2956,19 @@ exports.commands = {
 			Ladders(target).getTop().then(result => {
 				connection.send('|queryresponse|laddertop|' + JSON.stringify(result));
 			});
+		} else if (cmd === 'userlist') {
+			if (!trustable) return false;
+
+			let userlist = [];
+
+			for (let i in room.users) {
+				if (!room.users[i].named) {
+					continue;
+				}
+				userlist.push(room.users[i].userid);
+			}
+
+			connection.send(room + '|queryresponse|userlist|' + JSON.stringify(userlist));
```

Comment:
imo it should be ``|queryresponse|' + room + '|userlist| ...``

not that it matters much